httomo/wrappers_class.py

```diff
@@ -20,8 +20,14 @@
         print("CuPy is installed but GPU device inaccessible")
 except ImportError:
     import numpy as xp
-
-    print("CuPy is not installed")
+    log_once(
+        f"CuPy is not installed",
```

Comment:
No interpolated variables so no f-string needed.
```suggestion
        "CuPy is not installed",
```

Reply:
Good point, I was being too willing to copy+paste and I missed that :sweat_smile: